### ISSUE TITLE
Adding version number into dist files

### ISSUE THREAD

**Issue by seahindeniz** [floating-ui/floating-ui#2480]
**Is your feature request related to a problem? Please describe.**
I'm unable to determine which version of Floating UI is used on DOM because the bundled files under dist directory doesn't contain a version info.
![image](https://github.com/floating-ui/floating-ui/assets/5789670/70106e6d-4d6b-4e6e-9239-fab2356a69c5)

**Describe the solution you'd like**
Adding the library name and it's current version in a comment that doesn't increase the size would be awesome. I'm not talking about going full advertising of course (like adding owner details, github source link etc.) A person can always search for a library by it's name and it's already sufficient enough

**Describe alternatives you've considered**
Adding version number into filename but that can impact apps that doesn't automatically handles libraries based on their version numbers. So it might be a breaking change.

**Comment by atomiks** [comment]
You can look at the version in `node_modules/@floating-ui/dom/package.json`, or the lockfile of your repo.

**Comment by seahindeniz** [comment]
No, I meant on production. Consider a case where your lib logic copies dist file to prod like a CDN server and use the file by adding it with a script tag into DOM

**Comment by FezVrasta** [comment]
Most package managers will automatically collect licenses from the different libraries and generate a file with all the licenses and the version.

**Comment by seahindeniz** [comment]
I believe UMD version are not handled by bundlers when your bundling process just copies the dist file from node_modules. Another case is, developer can download a file from CDN and then just put it in asset folder which will only get copied as an asset and not get bundled. In these case, developer won't add a version to its URL, and you'll have no way around to know the version

**Comment by atomiks** [comment]
It can be done with the `banner` option in Rollup. Shouldn't take much for a PR, though there are several packages to update and add it in.

```
/** @floating-ui/dom v1.5.3 (MIT) */
```

This is React's:

```
/**
 * @license React
 * react.production.min.js
 *
 * Copyright (c) Facebook, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
```

Their version exists at runtime but not in the banner.
